### DISCUSSION TITLE
onnxruntime.capi.onnxruntime_pybind11_state.InvalidGraph: [ONNXRuntimeError] : 10 : INVALID_GRAPH : Load model from model_fp16.onnx failed:This is an invalid model. Type Error: Type 'tensor(float16)' of input parameter (const_empty_float__132) of operator (Resize) in node (Resize__139) is invalid.

### DISCUSSION THREAD

**Discussion by 1615070057** [microsoft/onnxruntime#17221] [Other Q&A]
onnxruntime.capi.onnxruntime_pybind11_state.InvalidGraph: [ONNXRuntimeError] : 10 : INVALID_GRAPH : Load model from model_fp16.onnx failed:This is an invalid model. Type Error: Type 'tensor(float16)' of input parameter (const_empty_float__132) of operator (Resize) in node (Resize__139) is invalid.
Can you help me

**Comment by skottmckay** [comment]
Guessing your model is using fp16 for an input of the Resize node that does not support that data type. Please see the ONNX spec https://github.com/onnx/onnx/blob/main/docs/Operators.md#Resize and determine which input const_empty_float__132 is providing. 

If that is still a problem I'd suggest creating an Issue and providing the model and additional details (ORT version etc.) as requested. https://github.com/microsoft/onnxruntime/issues
